### ISSUE TITLE
Form fields lack accessible names

### ISSUE THREAD

**Issue by arethsu** [JohnXLivingston/peertube-plugin-livechat#709]
**Description of issue**

This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 4.1.2 Name, Role, Value (Level A)](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value).

Form fields must have accessible names which are announced to screen readers and other assistive technology, in addition to being visible on the screen for sighted users. Usually this is the same element, which it will be if you use the `<label>` element correctly.

In the chat widget, both the nickname field, the emoji chat field and the message textarea lack accessible names, which means that assistive tech users will have trouble identifying their purpose. All they have is a placeholder text, as seen below:

![No accessible name available on target, devtools screenshot.](https://github.com/user-attachments/assets/f7f4b61b-a595-4f2f-bd9c-f202f4b5b7de)

The nickname field does have a visible label next to it, but it's not programmatically connected to the field, so it's not really helpful to assistive technologies.

![Label available but not linked to field programatically, devtools screenshot.](https://github.com/user-attachments/assets/2845f57f-4be7-4f16-bf93-e51e833680d8)

The second nickname field that appears if one tries to use an occupied name has the same issue.

**To Reproduce**

1. Inspect message input and nickname input form fields

**Recommended solution**

It's usually best to use the `<label>` element to name form fields and link it to its associated field with the `for` attribute. It's also acceptable to use the `title` attribute on the form field if there is no visible field label but the context still makes the field's purpose clear to sighted users.

Code example 1:
```
<label for=""name"">Name</label>
<input id=""name"" type=""text"">
```

Code example 2:
```
<label>Name<input id=""name"" type=""text""></label>
```

Code example 3:
```
<label for=""search-term"">Search for:</label>
<input id=""search-term"" name=""search-term"" type=""text"" value="""">
<select title=""Search on"" id=""scope"">
  <option>This page</option>
  <option>Whole site</option>
</select>
```

**Server:**

 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: TBA

**Desktop:**
 - OS: macOS 15.5
 - Browser: Safari 18.5

**Additional context**

This issue was part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), under assignment of [HowlRound Theatre Commons](https://howlround.com/).

**Comment by JohnXLivingston** [comment]
Hi @arethsu and @Woebin 

Thanks for all your accessibility-related issues! I'll check them in a few days.

**Comment by JohnXLivingston** [comment]
@arethsu , thanks :)

I see that you were commissioned by theHowlRound Theatre Commons. Is there any funding for fixing your findings? (just asking)
